Modules/Core/Common/include/itkMacro.h

```diff
@@ -188,14 +188,14 @@ namespace itk
 #  error "The SGI compiler is not supported under ITKv4 and above"
 #endif
 #if defined(__APPLE__)
-#  if defined(__clang__) && (__cplusplus < 201103L)
-#    error "Apple LLVM < 5.0 (clang < 3.3) is not supported under ITKv5"
+#  if defined(__clang__) && (__cplusplus < 201402L)
+#    error "Apple LLVM < 5.1 (clang < 3.4) is not supported under ITKv5.3"
```

Comment:
It appears that ITK.macOS.Python (Azure.Mac-1622555791099 Darwin-Build4705-PR2563-CXX_STANDARD-from-11-to-14-Python) fails here, because `defined(__clang__) && (__cplusplus < 201402L)`

At https://open.cdash.org/viewBuildError.php?buildid=7259248

Any suggestion? Is the compiler properly set to C++14 for ITK.macOS.Python?

Reply:
@N-Dekker this may need to be updated:

https://github.com/InsightSoftwareConsortium/ITK/blob/71bb6a99fe02fe80e8e39ed4e00b009b3c7444dd/Wrapping/Generators/CastXML/CMakeLists.txt#L220-L224